app/src/main/kotlin/com/wire/android/ui/common/WireLabelledCheckbox.kt

```diff
@@ -0,0 +1,50 @@
+package com.wire.android.ui.common
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.requiredHeight
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Checkbox
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.wire.android.ui.theme.wireDimensions
+import com.wire.android.ui.theme.wireTypography
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun WireLabelledCheckbox(
+    label: String,
+    checked: Boolean,
+    onCheckClicked: ((Boolean) -> Unit),
```

Comment:
I'm wondering if it would be better to have `onCheckedChanged` - update the `checked` value inside this item and just emit the current updated value instead of "click event" where we have to update it from outside

Reply:
I think it could be a good approach also, but I think I'll leave it as it is right now, because the current usages are depending on this callback, I'll mark a `// TODO` to improve, and we can come back later if we need another usage type for example